### PR TITLE
more helpful defaults & index.html 

### PR DESCRIPTION
**Pull request by kylemac**
- add more helpful blocks to layout
- attribute `spike`
- arial/helvetica font by default
- add info about the stack in index.html

screen shot 2016-05-17 at 2 58 49 pm

### CODE REVIEW

File: root/views/layout.jade [hunk 1]
```diff
@@ -2,16 +2,22 @@ doctype html
...
-    meta(name='author', content= "author of the site")
-    //- uncomment if your site is responsive!
-    //- meta(name="viewport" content="width=device-width, initial-scale=1")
-    title Title of Your Site
-    link(rel='stylesheet', href='/css/index.css')
+    block meta
+      meta(charset='utf-8')
+      meta(http-equiv='X-UA-Compatible', content='IE=edge, chrome=1')
+      meta(name='description', content='')
+      meta(name='author', content= "<%= github_username %>")
```
Comment (jescalan) [inline root/views/layout.jade]: What if we add a `meta(name='built-with', content='spike')`? Is that legit?

Comment (kylemac) [inline root/views/layout.jade]: naw, that seems greedy - we don't need that type of attribution

**Comment by kylemac** [comment]
@jescalan i'm thinking we might also want to link to the medium.com/static-dev (once its up) so new users can view how-tos of some good use-cases.

anything else you think we should add to this?

**Comment by jescalan** [comment]
Nope, this looks good to me! We can link the medium page now, it should be back up soon, their tech team is looking into it as of last night 😁

**Pull request merged**